aws/registers/templates/fluentd.conf

```diff
@@ -52,4 +52,13 @@
     endpoint https://endpoint1.collection.eu.sumologic.com/receiver/v1/http/${sumologic_key}
     log_format json
   </store>
+
+  <store>
+    type logit
+    stack_id ${logit_stack_id}
+    port 16081
```

Comment:
Ahhh, I didn't realise this. I clearly made a bunch of assumptions about this being the `TCP-SSL` port. I'll extract it to some configuration.